src/smriprep/cli/run.py

```diff
@@ -127,11 +136,14 @@ def _drop_ses(value):
     )
     g_bids.add_argument(
         '--anat-reference-method',
-        choices=['robust-template', 'session'],
-        default='robust-template',
-        help='Method to produce the reference anatomical space. '
-        '"robust-template" will data across sessions.'
-        '"session" will process each session individually.',
+        choices=['first', 'robust-template', 'session'],
+        default='first',
+        help='Method to produce the reference anatomical space:'
+        '\t"first" will use the first alphabetically sorted image'
+        '\t"robust-template" will construct an unbiased template from all images '
+        '(previously "--longitudinal")'
+        '\t"session" will independently process each session. If multiple sessions are '
+        'found, the behavior will be similar to "first"',
```

Comment:
How would you feel about `--subject-anatomical-reference {first-alphabetically,unbiased,sessionwise}` instead? That would match QSIPrep's terminology and we wouldn't have to update QSIPrep to match sMRIPrep.

Reply:
I like `unbiased` and `sessionwise`, but `first-alphabetically` feels a bit verbose...if `first` is too vague and you are still open to change, how about `first-abc` or `first-alpha`?

Reply:
`first-alpha` sounds good to me. @mattcieslak WDYT?